builder/data.py

```diff
@@ -37,8 +37,8 @@ class PKG_TOOLS(Enum):
 
     # where the cmake binaries should be stored, and dependencies installed
     'build_dir': 'build',
-    'deps_dir': 'build/deps',
-    'install_dir': 'build/install',
+    'deps_dir': '{build_dir}/deps',
```

Comment:
This allows the default deps_dir and install_dir to be based on the build_dir if it is specified in config, but they are not.